app/views/miq_ae_class/_class_fields.html.haml

```diff
@@ -61,6 +61,7 @@
                       = password_field_tag("fields_password_value_#{i}", '',
                         :placeholder       => placeholder_if_present(default_value),
                         :style             => field['datatype'] == "password" ? "" : "display:none",
+                        :autocomplete      => "new-password",
```

Comment:
I'm thinking we should just turn autocomplete off.  The reason is that it's unlikely that an automate password is something that someone would want to save in their browser and then fill in later.  That's usually reserved for the main login to the application, but here I don't think it makes sense.

Reply:
I added this because there is auto complete for password datatypes right now. Once I added this the auto complete went away.

Reply:
yes, new-password is definitely better than nothing., however, I was more thinking that `off` might be an even better choice.  See https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete for more details on the values.

Reply:
Then again https://stackoverflow.com/questions/15738259/disabling-chrome-autofill seems to be saying that Chrome is ignoring the value